app/mailers/devise_mailer.rb

```diff
@@ -22,4 +22,51 @@ def password_change(record, opts={})
 
     mail(to: record.email)
   end
-end
\ No newline at end of file
+
+  def invitation_instructions(record, token, opts={})
+    case record.team_members.last.role
+
+    when 'admin'
+      set_template('cca89657-03e4-4c88-9c9b-affd4c406ce6')
```

Comment:
Its more of a way to maintain the secrecy of things that we use. i.e. if someone gets access to our notify keys and know the notify template id's because we've published them openly on github then they can pretend to be us. 

Theres a reason the template is called `cca89657-03e4-4c88-9c9b-affd4c406ce6` rather than `invite-admin-user`